pkg/tfgen/generate_schema_test.go

```diff
@@ -368,3 +368,28 @@ func TestPropagateLanguageOptions(t *testing.T) {
 		assert.Equal(t, "gradle", actual["buildFiles"])
 	})
 }
+
+func TestDefaultInfoFails(t *testing.T) {
+	provider := testprovider.ProviderDefaultInfo()
+	meta, err := metadata.New(nil)
+	require.NoError(t, err)
+	provider.MetadataInfo = &tfbridge.MetadataInfo{
+		Path: "non-nil",
+		Data: meta,
+	}
+	err = provider.ApplyAutoAliases()
+	require.NoError(t, err)
+	defer func() {
+		r := recover()
```

Comment:
This is interesting! Admire getting it to work. Style nit: do any of require.Panicsf, require.Errorf, etc functions from the testing lib work here, can it make the code shorter?

Reply:
great suggestion, I wasn't aware of all the helper functions in that module, thanks!

Reply:
I would look at `assert.PanicsWithValue` or `assert.PanicsWithError`.